main.py

```diff
@@ -1,10 +1,46 @@
+from django.db import transaction
+
 import init_django_orm  # noqa: F401
 
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as f:
+        players_data = json.load(f)
+
+    with transaction.atomic():
+        for nickname, player_data in players_data.items():
+            race_data = player_data["race"]
+            race, _ = Race.objects.get_or_create(
+                name=race_data["name"],
+                defaults={"description": race_data.get("description", "")}
+            )
+
+            for skill_data in race_data.get("skills", []):
+                Skill.objects.get_or_create(
+                    name=skill_data["name"],
+                    race=race,
+                    defaults={"bonus": skill_data["bonus"]}
+                )
+
+            guild = None
+            if player_data.get("guild"):
+                guild_data = player_data["guild"]
+                guild, _ = Guild.objects.get_or_create(
+                    name=guild_data["name"],
+                    defaults={"description": guild_data.get("description")}
+                )
+            Player.objects.get_or_create(
+                nickname=nickname,
```

Comment:
The `nickname` field in the `Player` model should handle unique constraints. If a player with the same nickname already exists, consider handling this scenario to prevent integrity errors.